### DISCUSSION TITLE
Fallback to global

### DISCUSSION THREAD

**Discussion by jasonvarga** [open-southeners/vscode-laravel-pint#14] [Ideas]
Would you be open to falling back to the global `pint` command if it's not in the local composer.json?

We have a package that we'd like to use Pint on, but composer requiring it would force the minimum PHP version to 8.0, which we don't want to do yet. In our case, using the global `pint` command works fine.

**Comment by d8vjork** [comment]
I see this reasonable, I'll take a look on this later

**Reply by d8vjork** [reply]
Anyway, meanwhile, you can configure the executable at the User or Workspace part of VS Code in the extension settings

**Reply by jasonvarga** [reply]
That's good enough for me. I didn't realize there was a config option. 👍

**Reply by d8vjork** [reply]
Comment to here that the new pre-release has this feature among other features and fixes, setting the option `laravel-pint.fallbackToGlobalBin` which is enabled by default:

<img width="668" alt="image" src="https://user-images.githubusercontent.com/2331052/182810832-5118562e-552b-4e42-829f-2646d46a049b.png">

**Reply by jasonvarga** [reply]
Thank you!
